@garymathews , I am getting this different error with the fix.

```
[ERROR] :  Failed to compile Java source files:
[ERROR] :  
[ERROR] :  /Users/lchoudhary/Desktop/workspaces/workspace_2018/TIMOB-25749/build/android/gen/com/app/timob25749/Timob25749Application.java:113: error: cannot find symbol
[ERROR] :  						(Class<KrollSourceCodeProvider>) Class.forName(className));
[ERROR] :  						       ^
[ERROR] :    symbol:   class KrollSourceCodeProvider
[ERROR] :    location: class Timob25749Application
[ERROR] :  Note: Some input files use unchecked or unsafe operations.
[ERROR] :  Note: Recompile with -Xlint:unchecked for details.
[ERROR] :  1 error
[ERROR] Application Installer abnormal process termination. Process exit value was 1
```

--- a/android/templates/build/App.java
+++ b/android/templates/build/App.java
@@ -8,6 +8,7 @@ package <%= appid %>;
 import org.appcelerator.kroll.runtime.v8.V8Runtime;
 
 import org.appcelerator.kroll.KrollExternalModule;
+import org.appcelerator.kroll.common.KrollSourceCodeProvider;
 import org.appcelerator.kroll.KrollModule;
 import org.appcelerator.kroll.KrollModuleInfo;
 import org.appcelerator.kroll.KrollRuntime;
@@ -62,7 +63,7 @@ public final class <%= classname %>Application extends TiApplication
 			try {
 				runtime.addExternalCommonJsModule(
 						"<%- module.manifest.moduleid %>",
-						(Class<KrollExternalModule>) Class.forName(className));
+						(Class<KrollSourceCodeProvider>) Class.forName(className));
 			} catch (Throwable ex) {
 				Log.e(TAG, "Failed to add external CommonJS module: " + className);
 				if ((ex instanceof RuntimeException) == false) {

